### PR TITLE
Update linuxkit/init to gettyless; add getty to relevant examples

### PR DESCRIPTION
**Pull request by deitch**
Signed-off-by: Avi Deitcher <avi@deitcher.net>

**- What I did**

Updated `linuxkit/init` to the gettyless version per #1978  and inserted `linuxkit/getty` as a `service` per #1977 for relevant examples. 

**- How I did it**

Updated the yml files.

**- How to verify it**

Build and run them.

**- Description for the changelog**

Update init in images to use gettyless and use getty where relevant.

**- A picture of a cute animal (not mandatory but encouraged)**

Per the request on the other PRs to update examples.

### CODE REVIEW

**Comment by deitch** [comment]
**Note:** Recommend holding off merging this until the minor issue fixed in https://github.com/linuxkit/linuxkit/pull/2005 is done. Once that is merged, and new `linuxkit/init` hash is in hub, update this one before merging.

File: examples/gcp.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ kernel:
   image: "linuxkit/kernel:4.9.x"
   cmdline: "console=ttyS0 page_poison=1"
 init:
-  - linuxkit/init:1b8a7e394d2ec2f1fdb4d67645829d1b5bdca037
+  - linuxkit/init:502b12be51753f7910e7362dad3e5f4ea77646c5
   - linuxkit/runc:3a4e6cbf15470f62501b019b55e1caac5ee7689f
   - linuxkit/containerd:b50181bc6e0084e5fcd6b6ad3cf433c4f66cae5a
```
Comment (justincormack) [inline examples/gcp.yml]: gcp has an accessible console so adding a getty to the example is useful.

Comment (deitch) [inline examples/gcp.yml]: Forgot about that. Updated.

File: examples/redis-os.yml [hunk 2]
```diff
@@ -21,6 +21,10 @@ services:
      - CAP_SETGID
      - CAP_DAC_OVERRIDE
     net: host
+  - name: getty
+    image: "linuxkit/getty:148946d72d1c96df3ea91cb8ee4f9583cd3cc5c2"
+    env:
```
Comment (justincormack) [inline examples/redis-os.yml]: again, at the top

File: examples/node_exporter.yml [hunk 3]
```diff
@@ -12,6 +12,10 @@ services:
     image: "linuxkit/dhcpcd:7d2b8aaaf20c24ad7d11a5ea2ea5b4a80dc966f1"
   - name: node_exporter
     image: "linuxkit/node_exporter:265b35e1c9bd659fff22b0345fc7c8735c787b3c"
+  - name: getty
+    image: "linuxkit/getty:148946d72d1c96df3ea91cb8ee4f9583cd3cc5c2"
```
Comment (deitch) [inline examples/node_exporter.yml]: OK, updating them now

File: examples/sshd.yml [hunk 4]
```diff
@@ -14,6 +14,10 @@ services:
     image: "linuxkit/rngd:1fa4de44c961bb5075647181891a3e7e7ba51c31"
   - name: dhcpcd
     image: "linuxkit/dhcpcd:7d2b8aaaf20c24ad7d11a5ea2ea5b4a80dc966f1"
+  - name: getty
+    image: "linuxkit/getty:148946d72d1c96df3ea91cb8ee4f9583cd3cc5c2"
+    env:
+     - INSECURE=true
   - name: sshd
```
Comment (justincormack) [inline examples/sshd.yml]: the sshd example probably doesnt need a getty? As you can already log in?

Comment (deitch) [inline examples/sshd.yml]: I prefer to leave it, if we can. If someone is logging in, let's give them the console as well.

**Comment by deitch** [comment]
All at the top, first service where relevant

**Comment by justincormack** [comment]
`linuxkit/init:2599bcd5013ce5962aa155ee8929c26160de13bd` is updated after other fix

**Comment by deitch** [comment]
Thanks. I'll update these.

**Comment by deitch** [comment]
OK updated. All yours

File: examples/vsudd.yml [hunk 5]
```diff
@@ -14,7 +14,7 @@ services:
     image: "linuxkit/vsudd:a66df914201aac784195d5b78262f622fe7f910c"
     binds:
         - /run/containerd/containerd.sock:/run/containerd/containerd.sock
-    command: ["/vsudd",
```
Comment (riyazdf) [inline examples/vsudd.yml]: thanks for this fix 👍

**Pull request merged**